src/components/ctf-components/ctf-text-banner/__generated/ctf-text-banner.generated.ts

```diff
@@ -0,0 +1,45 @@
+import * as Types from '../../../../lib/__generated/graphql.types';
+
+import { useQuery, UseQueryOptions } from '@tanstack/react-query';
+import { customFetcher } from '@/lib/fetchConfig';
+export type TextBannerFieldsFragment = { __typename: 'TextBanner', title?: string | null, description?: string | null, sys: { __typename?: 'Sys', id: string } };
```

Comment:
If you have ? you don't need to specify null anymore I thinkl

Reply:
The `?` in typings means the property itself is optional, and only applies to undefined (property doesn't exist), not to null (since null is a value). So the typing is indeed correct if null is an allowed value.
However, when used in optional chaining, this does cover null values as well, maybe that's why this may seem confusing?